src/Nethermind/Nethermind.Store/DbExtensions.cs

```diff
@@ -74,11 +74,12 @@ public static void Set(this IDb db, long key, byte[] value)
             db[key.ToBigEndianByteArrayWithoutLeadingZeros()] = value;
         }
         
-        public static byte[] Get(this IDb db, long key)
-        {
-            return db[key.ToBigEndianByteArrayWithoutLeadingZeros()];
-        }
-        
+        public static byte[] Get(this IDb db, long key) => db[key.ToBigEndianByteArrayWithoutLeadingZeros()];
+
+        public static byte[] ToDbKey(this long key) => key.ToBigEndianByteArrayWithoutLeadingZeros();
+
+        public static byte[] ToDbKey(this Keccak key) => key.Bytes;
```

Comment:
I would prefer not to add these extensions here <- these are approaches used by specific dbs